src/BackpackServiceProvider.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Backpack\CRUD;
 
+use Backpack\Basset\BassetManager as Basset;
```

Comment:
@tabacitu it could be because those tests are not loading the needed providers `BassetServiceProvider` ? 

https://github.com/Laravel-Backpack/CRUD/blob/14ee61adcf9793d0a8974d773cdafdc7287cd1da/tests/BaseTest.php#L31

Cheers

Reply:
That's EXACTLY why! Thanks Pedro, that solves it!